ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,407 @@
+  const [nihFunded, setNihFunded] = useState<boolean>(null);
 
-        <ModalFooter style={{marginTop: 0, justifyContent: 'flex-start'}}>
-          <FlexColumn>
-            <TextColumn>
-              <p style={{marginTop: 0}}>Don't already have a Google Cloud account?</p>
-              <a href='https://cloud.google.com' target='_blank'>Create an account</a>
-            </TextColumn>
+  const validatePhoneNumber = (phoneInput: string) => {
+    const sanitizedPhone = phoneInput.replace(/\D/g, '');
 
-            <TextColumn>
-              <div>Add [ {this.props.profileState.profile.username} ] as a 'User' to your Google Cloud billing account.</div>
-              <a href='https://console.cloud.google.com/billing' target='_blank'>Add user</a>
-            </TextColumn>
-          </FlexColumn>
-        </ModalFooter>
+    if (sanitizedPhone.length >= 10 && sanitizedPhone.length <= 11) {
+      setInvalidPhoneNumberInput(false);
+      setPhoneNumber(sanitizedPhone);
+    } else {
+      setInvalidPhoneNumberInput(true);
+    }
+  };
 
-        <FontAwesomeIcon
-          icon={faTimes}
-          size='lg'
-          style={{
-            color: colors.accent,
-            position: 'absolute',
-            top: '1rem',
-            right: '1rem',
-            cursor: 'pointer'
-          }}
-          onClick={() => this.props.onClose()}
-        />
-      </Modal>;
+  const sendCreateBillingEmail = fp.flow(
+    withErrorModal({
+      title: 'Failed To Send Email',
+      message: 'An error occurred trying to send email. Please try again.',
+    })
+  )(async() => {await profileApi().sendBillingSetupEmail(
+    {
+      phone: phoneNumber,
+      paymentMethod: useCreditCard ? BillingPaymentMethod.CREDITCARD : BillingPaymentMethod.PURCHASEORDER,
+      isNihFunded: nihFunded,
+      institution: verifiedInstitutionalAffiliation.institutionDisplayName
     }
-  }
-);
+  );
+    setCurrentStep(4);
+  });
+
+  return <Modal width={650} onRequestClose={() => onClose()}>
+        <ModalBody style={{marginTop: '-4'}}>
+          <FlexColumn>
+            <FlexRow style={{alignItems: 'center', width: '620px', marginBottom: '0.8rem'}}>
+              <GoogleCloudLogoSvg style={{height: '33px', width: '207px', marginLeft: '-0.5rem', marginRight: '0.5rem'}}/>
+              <div style={styles.line}></div>
+              <div style={{paddingTop: 5, marginLeft: '1rem', marginRight: '2rem'}}>
+                <div style={styles.textHeader}>Create billing account</div>
+              </div>
+              {fp.range(1, 3).map((i) => <div style={stylesFunction.stepButtonCircle(currentStep, i)}>{i}</div>)}
+            </FlexRow>
+            {currentStep === 0 && <TextColumn>
+              <div style={styles.textNormal}>Billing accounts are managed via Google Cloud Platform™ service.</div>
+              <div style={styles.textNormal}><a href={supportUrls.createBillingAccount}>Learn more</a>
```

Comment:
Space is missing after the link 

![image](https://user-images.githubusercontent.com/822298/130281629-4ddf81e4-fbf2-4e04-a36a-370851c4e1b2.png)